web/pages/homepage.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html>
```

Comment:
Nice start! Now we just need to convert this file into a React component. 
Check out pages/index.js for example! You can actually replace the JSX from within this file with the content you've made here, with some slight modifications.

JSX is very similar to HTML, but has a few minor differences. I'm not going to try to explain the differences, but this video at the linked time should give a decent explaination https://youtu.be/mTz0GXj8NN0?t=697

Reply:
Okay! I’ll get ‘er converted tonight